src/components/Header/Header.test.tsx

```diff
@@ -68,7 +68,7 @@ describe('Header component', () => {
 
     expect(nav).toHaveClass('is-visible')
 
-    await user.click(screen.getByRole('button', { name: 'close' }))
+    await user.click(screen.getByRole('button', { name: 'Close' }))
```

Comment:
I'm not sure why this test decided to start failing for me, but it seemed to be a case-sensitivity error